### PR TITLE
Updated tests for suppression reason description

### PR DESCRIPTION
**Pull request by jayasreemca**
Update tests for suppression reason description for IFS and SOL

### CODE REVIEW

File: src/test/resources/features/ifs/Supression.feature [hunk 1]
```diff
@@ -2,8 +2,8 @@ Feature: Suppression
 
   Scenario: Suppression applied to main trans
     Given suppression data has been created
-      | reason | enabled | fromDate   | toDate     |
-      | POLICY | true    | 2021-04-04 | 2021-05-04 |
+      | reason | description | enabled | fromDate   | toDate     |
+      | POLICY | POLICY      | true    | 2021-04-04 | 2021-05-04 |
```
Comment (roccobruno) [inline src/test/resources/features/ifs/Supression.feature]: use a different value from the reason one please

File: src/test/resources/features/ifs/Supression.feature [hunk 2]
```diff
@@ -23,15 +23,15 @@ Feature: Suppression
...
+      | 2021-02-01 | 2021-04-03 | 61           | 2.6          | 35                      | 502172             |        |      |             |
+      | 2021-04-04 | 2021-05-04 | 31           | 0.0          | 0                       | 500000             | POLICY | 1    | POLICY      |
+      | 2021-05-05 | 2021-07-06 | 63           | 2.6          | 35                      | 502243             |        |      |             |
 
   Scenario: Suppression, 2 payments on same day during suppression
     Given suppression data has been created
-      | reason      | enabled | fromDate   | toDate     |
-      | LEGISLATIVE | true    | 2021-04-04 | 2021-05-04 |
+      | reason      | description | enabled | fromDate   | toDate     |
+      | LEGISLATIVE | LEGISLATIVE | true    | 2021-04-04 | 2021-05-04 |
```
Comment (roccobruno) [inline src/test/resources/features/ifs/Supression.feature]: different value from reason please

File: src/test/resources/features/ifs/Supression.feature [hunk 3]
```diff
@@ -54,17 +54,17 @@ Feature: Suppression
...
+      | 2021-02-01 | 2021-04-03 | 61           | 2.6          | 24                      | 351520             |             |      |             |
+      | 2021-04-04 | 2021-05-04 | 31           | 0.0          | 0                       | 350000             | LEGISLATIVE | 1    | LEGISLATIVE |
+      | 2021-05-05 | 2021-07-06 | 63           | 2.6          | 24                      | 351570             |             |      |             |
 
   Scenario: Suppression, 2 duties, 2 payments on same day for one of the duties
     Given suppression data has been created
-      | reason      | enabled | fromDate   | toDate     |
-      | LEGISLATIVE | true    | 2021-04-04 | 2021-05-04 |
+      | reason      | description | enabled | fromDate   | toDate     |
+      | LEGISLATIVE | LEGISLATIVE | true    | 2021-04-04 | 2021-05-04 |
```
Comment (roccobruno) [inline src/test/resources/features/ifs/Supression.feature]: same as before

File: src/test/resources/testdata/suppression/suppressionData.txt [hunk 4]
```diff
@@ -1,6 +1,7 @@
      {
           "code": "<REPLACE_code>",
           "reason":"<REPLACE_reason>",
+          "description":"<REPLACE_reason>",
```
Comment (roccobruno) [inline src/test/resources/testdata/suppression/suppressionData.txt]: should this be REPLACE_description ??

**Review by roccobruno — COMMENTED** [review]
Can u use different values for the description please?? Otherwise bugs can slip in..

**Review by thadeusssewanyana — APPROVED** [review]

File: src/test/resources/testdata/suppression/suppressionData.txt (hunk 4 above)
Comment (jayasreemca) [reply src/test/resources/testdata/suppression/suppressionData.txt]: implemented all comments.

**Comment by jayasreemca** [comment]
> Can u use different values for the description please?? Otherwise bugs can slip in..

Cool, changed to differnt value.

**Pull request merged**